docs/sections/user_guide/cli/drivers/make_solo_mosaic.rst

```diff
@@ -35,6 +35,15 @@ Its contents are described in section :ref:`make_solo_mosaic_yaml`.
 
   The driver creates a ``runscript.make_solo_mosaic`` file in the directory specified by ``run_dir:`` in the config and runs it, executing ``make_solo_mosaic``.
 
+..
+  An example runscript:
+
+  .. literalinclude:: make_solo_mosaic/runscript.cmd
+     :language: text
+     :emphasize-lines: 5
+  .. literalinclude:: make_solo_mosaic/runscript.out
+     :language: text
```

Comment:
This block is currently commented out because the prototype `docs/shared/make_solo_mosaic.yaml` appears not to be valid (or its schema is incorrect):
```
% uw make_solo_mosaic validate --config-file docs/shared/make_solo_mosaic.yaml 
[2024-06-15T15:25:22]     INFO Validating config against internal schema make-solo-mosaic
[2024-06-15T15:25:22]    ERROR 2 UW schema-validation errors found
[2024-06-15T15:25:22]    ERROR Error at make_solo_mosaic -> config:
[2024-06-15T15:25:22]    ERROR   Additional properties are not allowed ('tile_file' was unexpected)
[2024-06-15T15:25:22]    ERROR Error at make_solo_mosaic -> config -> mosaic_name:
[2024-06-15T15:25:22]    ERROR   'ocean_mosaic' is not of type 'integer'
[2024-06-15T15:25:22]    ERROR YAML validation errors
```
We could fix this later and uncomment this block (remove `..` line 38).